### ISSUE TITLE
Multi-GPU training

### ISSUE THREAD

**Issue by akshu281** [ZhihengCV/Bayesian-Crowd-Counting#31]
Hello,

Thank you for the open-sourced code, I am trying to replace the current backbone with a bigger network for my own dataset. However I cannot run this in single GPU. Could you please let me know how to achieve multi-gpu training in this case? Is it different from the other approaches that follow multigpu using dataparallel ?

Thank you in advance for this help !

**Comment by raisinglc** [comment]
I add more gpu_id to device setting, change worker number and using torch.DataParallel to map the model to different gpus, This is the simplest way. You could find more detail at here: 
https://pytorch.org/tutorials/intermediate/ddp_tutorial.html?highlight=torch%20dataparallel
https://pytorch.org/tutorials/beginner/blitz/data_parallel_tutorial.html?highlight=parallel

**Comment by akshu281** [comment]
Thank you for your response @raisinglc ! I tried to follow the same methods, however, during the loss calculation, I encounter an issue. Have you experienced this before ?

**Comment by raisinglc** [comment]
What kind of issue did you encounter?  An error or the result of loss is incorrect? Are you training on a single machine with multiple GPUs (torch.DataParallel) or multiple machine with multiple GPUs(torch.DistributedDataParallel)? 

I didn't change the code too much and trained on a single machine with multi. gpus. I haven't encounter any issue.
